sys/linux/fs_ioctl.txt

```diff
@@ -48,6 +51,12 @@ type fscrypt_policy_mode_t[CONTENTS, FILENAMES] {
 
 fs_policy_flags = FS_POLICY_FLAGS_PAD_4, FS_POLICY_FLAGS_PAD_8, FS_POLICY_FLAGS_PAD_16
 
+fsverity_digest {
+	digest_algorithm	int16
```

Comment:
This is one of FS_VERITY_ALG_SHA256/FS_VERITY_ALG_SHA512/FS_VERITY_ALG_CRC32C? If yes, it's better to use `flags` type for this.